packages/compass/src/main/menu.ts

```diff
@@ -452,42 +458,39 @@ function darwinMenu(
   menuState: WindowMenuState,
   app: typeof CompassApplication
 ): MenuItemConstructorOptions[] {
-  const menu: MenuTemplate = [darwinCompassSubMenu(menuState, app)];
-
-  menu.push(connectSubMenu(false, app));
-  menu.push(editSubMenu());
-  menu.push(viewSubMenu(app));
-
-  if (menuState.showCollection) {
-    menu.push(collectionSubMenu(menuState.isReadOnly, app));
-  }
-
-  menu.push(windowSubMenu(app));
-  menu.push(helpSubMenu(menuState, app));
-
-  return menu;
+  return [
+    darwinCompassSubMenu(menuState, app),
+    connectSubMenu(false, app),
```

Comment:
Would be nice to move all feature specific menus to their locations in the app, but that's already a good start